src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Fixed an issue where the activity instance returned by `Activity.Current` was
```

Comment:
Fixed Issue [#4466](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4466), where the activity instance returned by `Activity.Current` was
different than instance obtained from `IHttpActivityFeature.Activity`.
[#5136](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5136)